CHANGELOG.md

```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.1] Preview Release
+### Added
+- Added support for NTLM Authentication [#998](https://github.com/Microsoft/mssql-jdbc/pull/998)
+- Added new connection property 'useFmtOnly' to retrieve parameter metadata [#1044](https://github.com/Microsoft/mssql-jdbc/pull/1044)
```

Comment:
'useFmtOnly' => `useFmtOnly`?

Reply:
UseFmtOnly is connection property (a string) not a source code API. ` is only meant for Code insertion, e.g. driver methods/class names/APIs